benchmark/FindCXXStandard.cmake

```diff
@@ -16,34 +16,42 @@ file(WRITE "${SOURCE_FILE}"
     "int main() { }                                                 \n"
 )
 
-# Check for C++14
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+message(STATUS "Check for C++14")
```

Comment:
The idea was just to have CMake report which standard it detected.
I'm ok either way.